docs/build-configuration/environment-configuration.md

```diff
@@ -21,10 +21,20 @@ allowAccessWhileBuilding: true
 
 ## Basic Environment Authentication
 
-You can set up basic authentication for your Probo environments with `basicAuth`. This will require users to enter the correct username and password to access the Probo site.
+You can set up Basic Authentication for your Probo builds with `basicAuth`. This will require users to enter the correct username and password to access the Probo build site.
 
 {% highlight yaml %}
 basicAuth:
   username: foo
   password: bar
 {% endhighlight %}
+
+## Modify PHP Environment Options in a Build
```

Comment:
@sk33lz Calling out that the `phpIniOptions` can be paired with Drupal and Wordpress in this paragraph might be accidentally misleading. I feel like it implies it _only_ works for these types of sites. I am leaning towards removing this last sentence. What do you think?

Reply:
Updated the language to be a bit more generic to just point to the plugins page.